mc2/views.py

```diff
@@ -35,6 +41,36 @@ def get_queryset(self):
         return self.get_controllers_queryset(self.request).order_by('name')
 
 
+class CreateAccountView(FormView):
```

Comment:
Have a look at [django-registration](https://django-registration.readthedocs.io/en/2.2/quickstart.html#quickstart) for an easier way to setup registration.